plugins/VRPNPlugin/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2015-2018, EPFL/Blue Brain Project
+# All rights reserved. Do not distribute without permission.
+#
+# This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+
+set(VRPNPLUGIN_HEADERS VRPNPlugin.h)
```

Comment:
shoould be a complete top-level file imo

Reply:
what do you mean by that?

Reply:
this file cannot be used standalone as a top-level project: no `project()`, no find packages, no `include(Common)`. A plugin must be defined like that, as normally it would be in its repo.

Reply:
ok I see, but the deflect and rockets plugin are the same...

Reply:
they are not 'real' plugins yet, as they need stuff that is not available from the pluginAPI. The optix plugin/module is a better example.